x/authz/keeper/keeper.go

```diff
@@ -237,14 +237,20 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) *authz.GenesisState {
 // InitGenesis new authz genesis
 func (k Keeper) InitGenesis(ctx sdk.Context, data *authz.GenesisState) {
 	for _, entry := range data.Authorization {
+		if entry.Expiration.Before(ctx.BlockTime()) {
+			continue
+		}
```

Comment:
Why do we continue here? I think we should return an error (or panic since we can't return).

Reply:
I was thinking more about. It's OK to continue - the grant will be valid but it will be pruned immediately... I would issue warning though.